templates/incl-editor-and-output.html

```diff
@@ -49,16 +50,10 @@
           class="flex-0 mt-0 bg-green-100 border-t-4 border-green-500 rounded-b text-green-900 px-4 py-3 shadow-md"
           role="alert" style="display: none;">
           <div class="flex">
-            <div class="py-1">
-              <svg class="fill-current h-6 w-6 text-green-500 mr-4" xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 20 20">
-                <path
-                  d="M2.93 17.07A10 10 0 1 1 17.07 2.93 10 10 0 0 1 2.93 17.07zm12.73-1.41A8 8 0 1 0 4.34 4.34a8 8 0 0 0 11.32 11.32zM9 11V9h2v6H9v-4zm0-6h2v2H9V5z" />
-              </svg>
-            </div>
-            <div>
+            <div class="flex" style="justify-content: space-between; width: 100%; align-items: center;">
               <p class="caption font-bold">Success</p>
-              <p class="details text-sm">Something went according to plan.</p>
+              <button onclick="hedyApp.confetti_cannon()">🎉</button>
```

Comment:
I think it is a bit non-obvious that clicking it will give confetti. Maybe adding a border would work?

Reply:
Nah I think it is clear enough! It is also an extra so if kids don't find it that is not al too bad (and they will all tell each other when one kid finds it 🤣 )